loopout.go

```diff
@@ -706,6 +706,50 @@ func (s *loopOutSwap) waitForHtlcSpendConfirmed(globalCtx context.Context,
 		return nil, fmt.Errorf("register spend ntfn: %v", err)
 	}
 
+	// Create a child context for a one time lookup of our payment.
+	// We defer cancelling it so that lnd will terminate the stream.
```

Comment:
Yes? We want to lnd that we no longer need this subscription once this function exits? Otherwise we'll have a bunch of unused track payment subscriptions in lnd.

Reply:
I mean, is `lnd` the entity that terminates the stream? I think our client is still terminating the stream at the end of the function?

Reply:
Ok, reusing the existing cancel context.